docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/jwt-authentication.md

```diff
@@ -0,0 +1,22 @@
+---
+title: JWT Authenticaiton
+---
+<!-- Edit Canonical Link -->
+<head> 
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/jwt-authentication"/>
+</head>
+
+Many 3rd party integrations available for Kubernetes, such as Gitlab and Hashicorp Vault, involve giving an external process access to the Kubernetes API using a native Kubernetes Service Account token for authentication.
+
+In Rancher v2.9.0 and later, you can enable a downstream cluster to support JWT authentication of tokens created for a service account on a downstream cluster through the Rancher authentication proxy. Prior to this feature, Rancher rejects such requests because Rancher only supports Rancher-issued tokens.
```

Comment:
```suggestion
In Rancher v2.9.0 and later, you can enable a downstream cluster to support JSON web token (JWT) authentication of tokens created for a service account on a downstream cluster through the Rancher authentication proxy. In Rancher versions earlier than  v2.9.0, only Rancher-issued tokens were supported.
```

I'm having some trouble making "you can enable a downstream cluster to support JSON web token (JWT) authentication of tokens created for a service account on a downstream cluster through the Rancher authentication proxy" shorter and easier to read. I feel like this should probably be split into two sentences, or reorganized so there aren't as many clauses. Something like, "service accounts on downstream clusters can now authenticate through JWT using the Rancher authentication proxy", if I'm understanding the sentence correctly.